com.github.AberothCommunity.Aberoth.metainfo.xml

```diff
@@ -31,6 +31,58 @@
   <categories>
     <category>Game</category>
   </categories>
+  <releases>
+    <release date="2023-7-2">
```

Comment:
You are missing the actual version from releases. This won't work

Reply:
Sorry about that, I forgot to validate